### PR TITLE
Optional SIPS when registering

### PR DESCRIPTION
**Pull request by lminiero**
See #375

### CODE REVIEW

File: plugins/janus_sip.c [hunk 1]
```diff
@@ -2450,18 +2467,32 @@ gpointer janus_sip_sofia_thread(gpointer user_data) {
...
-				SIPTAG_ALLOW_STR("INVITE, ACK, BYE, CANCEL, OPTIONS"),
-				NUTAG_M_USERNAME(session->account.username),
-				NUTAG_URL(sip_url),
-				NUTAG_SIPS_URL(sips_url),
-				SIPTAG_USER_AGENT_STR(user_agent),
-				NUTAG_KEEPALIVE(keepalive_interval * 1000),	/* Sofia expects it in milliseconds */
-				NUTAG_OUTBOUND(outbound_options),
-				SIPTAG_SUPPORTED(NULL),
-				TAG_NULL());
+	if(session->account.sips) {
```
Comment (saghul) [inline plugins/janus_sip.c]: I think you can simplify this by using TAG_IF: http://sofia-sip.sourceforge.net/refdocs/su/su__tag_8h.html#a8ccbc9cc9f07d5670065e990d07bb428

**Comment by saghul** [comment]
LGTM, but have a look at TAG_IF :-)

File: plugins/janus_sip.c (hunk 1 above)
Comment (lminiero) [inline plugins/janus_sip.c]: Oh, I wasn't aware of that, that's neat! I'll update the PR right away.

**Comment by saghul** [comment]
:+1:

**Comment by lminiero** [comment]
Thanks for the quick review, merging then!

**Comment by bhakimi** [comment]
just a note, the taf if still included the sips even when i did not make a sips request, i also posted these findings in the merged ticket

**Pull request merged**